src/nupic/research/connections.py

```diff
@@ -242,6 +259,48 @@ def updateSynapsePermanence(self, synapse, permanence):
     # Update indexes
     self._synapsesForPresynapticCell[newData.presynapticCell][synapse] = newData
 
+  def computeActivity(self, activeInput, activePermanenceThreshold,
```

Comment:
From the [style guide](https://github.com/numenta/nupic/wiki/Python-Style-Guide): "Separate methods inside classes by two blank lines."